plugins/powershell/requirements.txt

```diff
@@ -3,4 +3,5 @@
 # See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
 git+https://github.com/komand/pywinrm.git@v0.3.1dev0#pywinrm
 requests-kerberos==0.12.0
-requests-credssp==1.1.0
\ No newline at end of file
+requests-credssp==1.1.0
+parameterized      0.8.1
```

Comment:
why is there no "==" and what is parameterized here for? i dont see it being used elsewhere

Reply:
not sure why that happened